WebApiCoreSeed.UnitTests/Services/UserServiceTests.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.Threading.Tasks;
+using WebApiCoreSeed.Domain.Services;
+using WebApiCoreSeed.Domain.Services.Interfaces;
+
+namespace WebApiCoreSeed.UnitTests.Services
+{
+    [TestClass]
+    public class UserServiceTests : BaseServiceUnitTest
+    {
+        [TestMethod]
+        public async Task Should_Get_User_By_Id_Async()

```

Comment:
Checkout our [naming guide lines](https://github.com/MakingSense/code-style-guides/blob/master/CSharp/README.md#testing-code) for C# test code